app/controllers/application_controller.rb

```diff
@@ -5,6 +5,31 @@ class ApplicationController < Sinatra::Base
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "password_security"
+  end
+
+  get '/' do
+    if logged_in?
+      @user = current_user
```

Comment:
current_user is already available to us in the views. This is an unneeded instance variable.